File: Contests/Div4/817/A.py
def solve():
    n = int(input())
    s = input()
    nome = "Timur"
    nome = sorted(nome)
    nome = ''.join(nome)
    s = sorted(s)
    s = ''.join(s)
    print("YES" if s == nome else "NO")

t = int(input())
for i in range(t):
    solve()

File: Contests/Div4/817/A.cpp
#include <bits/stdc++.h>

using namespace std;

void solve(){
    int n;
    cin >> n;
    string s;
    cin >> s;
    string nome = "Timur";
    sort(nome.begin(), nome.end());
    sort(s.begin(), s.end());
    if (nome == s){
        cout << "YES" << '\n';
    } else cout << "NO" << '\n';
}   

int main(void){
    ios_base::sync_with_stdio(false); cin.tie(0);
    int t; cin >> t; while(t--){solve();}
}